### DISCUSSION TITLE
It doesn't feel good to see my post removed.

### DISCUSSION THREAD

**Discussion by amano-kenji** [jtdaugherty/brick#460] [General]
I understand you want to be the primary driver of your personal projects. I understand you have your own process and we have differences with regard to approaching a new feature like terminal image. I understand you have pressing concerns in your life as most people do. We all would rather not spend too much time on something that doesn't alleviate any personal or collective pain.

However, it doesn't feel good to see my post removed without a saying. It took a while to write that post. A better way was to lock the post and maybe state that you may do this in your own way when it becomes relevant for you.

I ask for not removing posts unless they are malicious. We all have personal issues. I'm just another flawed human being who may offend people from time to time without intending it.

**Comment by jtdaugherty** [comment]
Hi @amano-kenji - I'm sorry to hear that the post removal was concerning for you, and I can understand that it wasn't helpful that I didn't comment on why the post was removed.

The use of this Discussions feature on this repository is still new to me, and I am still figuring out how I want it to be used. What I am coming to discover is that I think it will be most useful as a `brick-users` email list alternative if it is a forum for discussions about Brick features, bugs, etc. I removed the post you are alluding to mainly for two reasons:

1. It was not about an existing or future Brick feature, but about a possible Vty feature. I did not want a post about Vty to be confusing in the Brick forum.
2. I had no intention of responding to it because we had already had lengthy discussion about the image feature elsewhere in at least one github issue. I did not want to continue that discussion here because I did not think it would be productive. I also did not want to leave the discussion open and unresolved in this forum because I was concerned that it would be confusing or misleading for other Brick users. The discussion was about a feature that was not likely be added to Vty or Brick any time soon, as I had hoped was clear from our discussion elsewhere.

With that said, I would like to see this forum used for questions about Brick and Brick application development, especially questions that might have answers that will benefit other Brick users. It would be helpful to me if we can conduct discussions about possible future features in Github issue comment threads on the appropriate repository.

I hope this helps clarify why I removed the post, and I also hope this helps clarify how I would like us to use this forum going forward. I would like to get your response, but I want to be sensitive to sharing personal concerns in this forum. If you would like to respond, please contact me directly at `cygnus AT foobox DOT com` and we can discuss further if that would be helpful. I'm going to lock this thread to leave this context up and to avoid hashing out any further details here.
